app/views/users/_form.html.erb

```diff
@@ -35,6 +35,9 @@
                         :label => _('Authorized by') } ) unless @editing_self %>
 
       <div id='password' <%= display? !@user.manage_password? %>>
+        <% if User.current == @user  %>
+          <%= password_f f, :current_password, :label => _('Current password') %>
```

Comment:
Should the placeholder be disabled here? The stored value is only really on the `:password` field, not current password.